Samples~/Cinemachine Example Scenes/Shared/Scripts/SpawnInCircle.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SpawnInCircle : MonoBehaviour
+{
+    public GameObject Prefab;
+    public float Radius = 1000;
+    public float Amount = 10000;
+    public bool DoIt;
+
+    void Update()
+    {
+        if (DoIt && Prefab != null)
+        {
+            var rootPos = transform.position;
+            var rot = transform.rotation;
+            for (int i = 0; i < Amount; ++i)
+            {
+                var a = Random.Range(0, 360);
+                var pos = new Vector3(Mathf.Cos(a), 0, Mathf.Sin(a));
+                pos = rootPos + pos * Mathf.Sqrt(Random.Range(0.0f, 1.0f)) * Radius;
+                Instantiate(Prefab, pos, rot);
```

Comment:
Instead of 
<pre>Instantiate(Prefab, pos, rot). </pre>
I would do 
<pre>
Instantiate(Prefab, pos, rot, <b>transform</b>). // this.transform
</pre>

This way the spawned objects don't make a mess in the editor hierarchy.